compiler/src/iree/compiler/Codegen/Common/GPU/GPUMaterializeEncoding.cpp

```diff
@@ -432,8 +432,16 @@ materializeFuncOpEncodings(FunctionOpInterface funcOp,
   MLIRContext *ctx = funcOp.getContext();
   {
     RewritePatternSet patterns(ctx);
-    MaterializeEncodingTypeConverter typeConverter(materializeEncodingForTarget,
-                                                   targetAttr);
+    // On GPU, we use transposeNarrowN=false for a combination of reasons:
+    // 1. As linalg.matmul materializes into iree_gpu.multi_mma, which inherits
+    //    its semantics from the wrapped intrinsic, we can't rely on any kind of
+    //    LHS<->RHS symmetry.
+    // 2. We do not currently use ukernels, which would be one of the main ares
+    //    to benefit from transposeNarrowN.
```

Comment:
Dumb question -- what is `ares`? Is it a typo?

Wiki told me that Ares is a the Greek god of war.

Reply:
I meant `areas`. Lol about ancient gods!